app/src/main/res/layout/dialpad.xml

```diff
@@ -60,9 +60,9 @@
             android:layout_below="@+id/dialpad_2"
```

Comment:
The attribute `android:layout_below` is not valid in the `LinearLayout`. It's there in every button from `dialpad_2_holder` to `dialpad_0_holder` except the asterisk.